### PR TITLE
New attributes added for integration testing

### PR DESCRIPTION
**Pull request by matiasmoreno876**
Hi team, in this PR I add new attributes in different buttons of the Wazuh menu, this is with the intention of making it easier to find the selector when performing integration tests with Cypress.

### CODE REVIEW

File: common/constants.ts [hunk 1]
```diff
@@ -204,7 +204,109 @@ export enum WAZUH_MODULES_ID{
...
+  GDPR = 'menu-modules-gdpr-link',
+}
+
+export enum WAZUH_MANAGEMENT_ID {
+  MANAGEMENT = 'management',
+  ADMINISTRATION = 'administration',
+  RULESET = 'ruleset',
+  RULES = 'rules',
+  DECODERS = 'decoders',
+  CDB_LISTS = 'lists',
```
Comment (MauGaP) [inline common/constants.ts:238]: could this id be `cdb-lists` for sake of consistency with the rest

File: public/components/wz-menu/wz-menu-management.js [hunk 2]
```diff
@@ -24,20 +25,76 @@ class WzMenuManagement extends Component {
...
-      statusReports: { id: 'statusReports', text: 'Status and reports' },
-      status: { id: 'status', text: 'Status' },
-      cluster: { id: 'monitoring', text: 'Cluster' },
-      logs: { id: 'logs', text: 'Logs' },
-      reporting: { id: 'reporting', text: 'Reporting' },
-      statistics: { id: 'statistics', text: 'Statistics' },
+      management: {
+        id: WAZUH_MANAGEMENT_ID.MANAGEMENT,
+        cyTestId: WAZUH_MANAGEMENT_CY_TEST_ID.MANAGEMENT,
+        text: 'Management',
```
Comment (MauGaP) [inline public/components/wz-menu/wz-menu-management.js:31]: do we need a comma after the last element of every block?

File: public/components/wz-menu/wz-menu-tools.js [hunk 3]
```diff
@@ -14,6 +14,8 @@ import { EuiFlexItem, EuiFlexGroup, EuiSideNav, EuiIcon } from '@elastic/eui';
 import { WzRequest } from '../../react-services/wz-request';
 import { connect } from 'react-redux';
 import { AppNavigate } from '../../react-services/app-navigate';
+import { WAZUH_TOOLS_ID, WAZUH_TOOLS_CY_TEST_ID } from '../../../common/constants';
+
```
Comment (MauGaP) [inline public/components/wz-menu/wz-menu-tools.js:18]: extra line

File: public/components/wz-menu/wz-menu.js [hunk 4]
```diff
@@ -710,6 +710,7 @@ export const WzMenu = withWindowSize(class WzMenu extends Component {
         <div className="wz-menu-left-side">
           <div className="wz-menu-sections" style={!thereAreSelectors ? { height: "100%" } : {}}>
             <EuiButtonEmpty
+                data-cy='menu-modules-button'
```
Comment (MauGaP) [inline public/components/wz-menu/wz-menu.js:713]: Is this the proper indentation?

File: public/components/wz-menu/wz-menu-management.js (hunk 2 above)
Comment (frankeros) [reply public/components/wz-menu/wz-menu-management.js:31]: this is considered good practice for some JS projects, we could choose if not.

File: common/constants.ts [hunk 5]
```diff
@@ -204,7 +204,109 @@ export enum WAZUH_MODULES_ID{
   TSC = 'tsc',
   CIS_CAT = 'ciscat',
   VIRUSTOTAL = 'virustotal',
-  GDPR = 'gdpr'
+  GDPR = 'gdpr',
+}
+
+export enum WAZUH_MODULES_CY_TEST_ID {
```
Comment (frankeros) [inline common/constants.ts:210]: It's not scalable, and this file is for business constants. 
I think this kind of constants should be in a different type of files, I suggest using a `<same-name-that-component>.cy.ts` file maybe for each tsx file or common resources.

File: common/constants.ts [hunk 6]
```diff
@@ -204,7 +204,109 @@ export enum WAZUH_MODULES_ID{
...
+  PCI_DSS = 'menu-modules-pci-link',
+  HIPAA = 'menu-modules-hipaa-link',
+  NIST_800_53 = 'menu-modules-nist-link',
+  TSC = 'menu-modules-tsc-link',
+  CIS_CAT = 'menu-modules-ciscat-link',
+  VIRUSTOTAL = 'menu-modules-virustotal-link',
+  GDPR = 'menu-modules-gdpr-link',
+}
+
+export enum WAZUH_MANAGEMENT_ID {
```
Comment (frankeros) [inline common/constants.ts:232]: the name of the constants are not easy to associate with his purpose, they are very generics

**Review by frankeros — CHANGES_REQUESTED** [review]

File: common/constants.ts (hunk 5 above)
Comment (Desvelao) [reply common/constants.ts:210]: Why we need these values defined in a file? There are unique, why not defined directly in the components?

Comment (matiasmoreno876) [reply common/constants.ts:210]: @frankeros I agree with you, it seems like a good idea to pass the constants corresponding to test in a different file.

Comment (matiasmoreno876) [reply common/constants.ts:210]: @Desvelao It seemed like a good idea to leave these constants outside the component, I tried to follow the idea that was used for the constants in 'modules'

File: public/components/wz-menu/wz-menu.js (hunk 4 above)
Comment (matiasmoreno876) [reply public/components/wz-menu/wz-menu.js:713]: There I check it 😄

File: common/constants.ts (hunk 6 above)
Comment (matiasmoreno876) [reply common/constants.ts:232]: Good observation there I check it.

**Review by frankeros — APPROVED** [review]
LGTM!

**Review by frankeros — APPROVED** [review]
LGTM!

**Review by gabiwassan — APPROVED** [review]
LGTM!

**Review by pablomarga — APPROVED** [review]
LGTM!

**Pull request merged**